wagtail/users/views/users.py

```diff
@@ -221,9 +254,6 @@ def get_base_queryset(self):
         else:
             users = User.objects.filter(self.group_filter)
 
-        if self.locale:
-            users = users.filter(locale=self.locale)
```

Comment:
This was added in f34b4be1646f108b2d23ef0e543af1ff9f15671c but I don't see how this would ever be used.